src/ebay-menu-button/menu-button.tsx

```diff
@@ -138,27 +152,29 @@ const EbayMenuButton: FC<Props> = ({
                     {label}
                 </EbayButton>
             }
-            <EbayMenu
-                baseEl="div"
-                ref={menuRef}
-                type={type}
-                className={menuClasses}
-                tabIndex={-1}
-                id={menuId}
-                autofocus
-                checked={checked}
-                onKeyDown={handleMenuKeydown}
-                onChange={onChange}
-                onSelect={onSelect}
-            >
-                {menuItems.map((item, i) =>
-                    cloneElement(item, {
-                        ...item.props,
-                        className: classnames(item.props.className, 'menu-button__item'),
-                        key: i
-                    })
-                )}
-            </EbayMenu>
+            {expanded &&
+                <EbayMenu
+                    ref={menuRef}
```

Comment:
we were passing DIV when we called Menu from menu-button.
https://github.com/eBay/ebayui-core-react/pull/220/files#diff-826c16984367073bbeea3c4c09175c16b98abada9a4452f4ba89aa01067148dfL142

@HenriqueLimas do u remember why this was the case?

Reply:
Good catch, I think it didn't render properly without the `div`